### PR TITLE
Enable xpu windows wheel smoke test

### PR DESCRIPTION
**Pull request by chuanqi129**
Works for https://github.com/pytorch/pytorch/issues/114850

### CODE REVIEW

File: .github/scripts/validate_binaries.sh [hunk 1]
```diff
@@ -6,6 +6,10 @@ else
     export PYTHON_RUN="python3"
     if [[ ${TARGET_OS} == 'windows' ]]; then
         export PYTHON_RUN="python"
+        # Currently xpu env need a helper script to activate
+        if [[ ${DESIRED_CUDA} == "xpu" ]]; then
```
Comment (atalman) [inline .github/scripts/validate_binaries.sh:10]:
```suggestion
        if [[ ${MATRIX_GPU_ARCH_TYPE} == "xpu" ]]; then
```

**Review by atalman — APPROVED** [review]
lgtm

File: .github/scripts/xpu_env_helper.bat [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+shift
+goto start
+
+:done
+if "%args%" == "" (
+    echo Usage: xpu_env_helper.bat [command] [args]
+    echo e.g. xpu_env_helper.bat icpx --version
+)
+
+%args% || exit /b 1
```
Comment (malfet) [inline .github/scripts/xpu_env_helper.bat:17]: What were the reason for using Batch script? Why can't it be a power shell one?

Comment (atalman) [reply .github/scripts/xpu_env_helper.bat:17]: cc @chuanqi129

**Pull request merged**